static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js

```diff
@@ -0,0 +1,79 @@
+define([
+    'commercial/modules/commercial-features',
+    'common/modules/commercial/contributions-utilities',
+    'lib/ajax',
+    'lib/config',
+    'lib/cookies',
+    'lib/storage',
+    'lodash/utilities/template',
+    'raw-loader!common/views/contributions-epic-single-button.html'
+], function (
+    commercialFeatures,
+    contributionsUtilities,
+    ajax,
+    config,
+    cookies,
+    store,
+    template,
+    contributionsEpicSingleButton
+) {
+
+    function canBeDisplayed() {
+        var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
+        var worksWellWithPageTemplate = (config.page.contentType === 'Article') && !config.page.isMinuteArticle; // may render badly on other types
+        var isSensitive = config.page.isSensitive === true;
+
+        return userHasNeverContributed &&
+            commercialFeatures.canReasonablyAskForMoney &&
+            worksWellWithPageTemplate && !isSensitive;
+    }
+
+    return contributionsUtilities.makeABTest({
+        id: 'EpicToSupportLandingPage',
+        campaignId: 'epic_to_support_landing_page',
+
+        start: '2017-03-29',
+        expiry: '2017-04-27',
+
+        author: 'Justin Pinner',
+        description: 'Use AB framework to push traffic to new supporter landing page.',
+        successMeasure: 'There is no noticeable drop-off in contributions.',
+        idealOutcome: 'Readers sign up for recurring monthly contributions in droves',
+
+        audienceCriteria: 'UK edition readers',
+        audience: 0.2,
+        audienceOffset: 0,
+        showForSensitive: true,
+        useTargetingTool: false,
+
+        overrideCanRun: true,
```

Comment:
This needs to be false in order to inherit from all the default behaviour. The canRun function bellow will still be taken into account. (see contributions-utilities.js:100)